IntegrationTests/UI.IntegrationTests/UITest.cs

```diff
@@ -135,6 +135,17 @@ public static void ProcessUntil(string processName, Func<bool> condition, int ma
             Assert.Fail($"'{processName}' didn't finish in {maxIterations} iterations");
         }
 
+        public static void ProcessEventsFor(int milliseconds)
+        {
+            const int delay = 25;
```

Comment:
Similar for line 132?
```suggestion
           // Same delay as RevisionDataGridView.BackgroundThreadUpdatePeriod
            const int delay = 25;
```